.editorconfig

```diff
@@ -41,6 +41,14 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
+[package.json]
```

Comment:
Did you intend for this to come in here?

Reply:
Yes, unfortunately the syntax doesn't allow for completely different paths for the same block, so it has to be duplicated.